packages/webpack-rtl-plugin/src/css-diff.js

```diff
@@ -0,0 +1,90 @@
+const merge = require( 'lodash.merge' );
```

Comment:
Here I would do `const { merge } = require( 'lodash' )`, i.e., use the full Lodash package that is almost certainly already installed.

Not a big issue, just a "import Lodash in a consistent way across the codebase" kind of improvement.

Reply:
OTOH it then depends on the bundler being able to do tree shaking to eliminate all the rest of lodash that's not being used. In a quick test, Webpack 5 at least doesn't seem to do so well about that with the code here. With a simple module like
```
const merge = require( 'lodash' );

module.exports = merge;
```
I get 70875 bytes for importing lodash, 14592 for importing lodash/merge, and 9646 for lodash.merge. Using esm syntax doesn't make much difference, probably since lodash itself isn't esm.

OTOH, this doesn't really need lodash's merge at all. The data structure is just objects and scalars, nothing very fancy.